app/views/fast/show.html.erb

```diff
@@ -0,0 +1,3 @@
+<% @suggestions.map do |suggestion| %>
+  <li class="list-group-item" role="option" data-autocomplete-value="<%= suggestion.values.first %>" data-autocomplete-label="<%= suggestion.keys.first %>"><%= html_escape(suggestion.keys.first) %></li>
```

Comment:
Can this be `<%= tag.li ...` instead of inlining variables into html attributes?

Reply:
You bet.